.github/workflows/ci.yml

```diff
@@ -123,6 +131,7 @@ jobs:
     env:
       LEGACY_CI: true
       JRUBY_OPTS: ${{ matrix.container.jruby_opts || '--dev' }}
+        NO_COVERAGE=true
```

Comment:
Note: this isn't actually impactful, since none of these were running it anyway. But I thought it would be clearer to supply it anyway - happy to drop it again if you disagree!